src/store/modules/Health/EventLogStore.js

```diff
@@ -0,0 +1,68 @@
+import api from '../../api';
+
+const EventLogStore = {
+  namespaced: true,
+  state: {
+    eventLogData: null
+  },
+  getters: {
+    eventLogData(state) {
+      return state.eventLogData;
+    }
+  },
+  mutations: {
+    setEventLogData(state, eventLogData) {
+      state.eventLogData = eventLogData;
+    }
+  },
+  actions: {
+    getEventLogData({ commit }) {
+      api
+        .get('/xyz/openbmc_project/logging/enumerate')
+        .then(response => {
+          const eventLog = response.data.data;
+          const entryNumber = /[1-9]/;
+          const eventLogEntries = [];
+          const severityToPriorityMap = {
+            Emergency: 'High',
+            Alert: 'High',
+            Critical: 'High',
+            Error: 'High',
+            Warning: 'Medium',
+            Notice: 'Low',
+            Debug: 'Low',
+            Informational: 'Low'
+          };
+          for (let key in eventLog) {
+            if (
+              key.includes('entry') &&
+              key.match(entryNumber) &&
+              !key.includes('callout')
```

Comment:
why are we excluding event logs that have a callout property ?

Reply:
the endpoints with the callout property do not have the log info that I need to display and throws an error